.eslintrc.js

```diff
@@ -15,6 +17,10 @@ module.exports = {
 
   globals: {
     __static: true,
+    ...globals.reduce((prev, curr) => {
+      prev[curr] = true;
```

Comment:
should it need to be `writable` instead of `true`
take a look at [specifying-globals](https://eslint.org/docs/user-guide/configuring#specifying-globals)
```suggestion
      prev[curr] = 'writable';
```